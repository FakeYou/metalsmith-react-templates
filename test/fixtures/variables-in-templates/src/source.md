---
title: This is a title
misc: Another key
---
Testing