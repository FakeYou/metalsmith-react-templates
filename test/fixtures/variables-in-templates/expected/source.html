<div><span>This is a title</span><span>Another key</span><span>Testing</span></div>